tests/test_e2e_05_topology.py

```diff
@@ -268,6 +284,65 @@ def test_060_enabling_all_interfaces_on_a_switch_persistent(self):
         for interface in data['switches'][switch_id]['interfaces']:
             assert data['switches'][switch_id]['interfaces'][interface]['enabled'] is True
 
+    def test_065_disabling_all_interfaces_on_a_switch_persistent(self):
```

Comment:
This test is doing enable and disable of the interface, even though the test name is: test_065_disabling_all_interfaces_on_a_switch_persistent. I suggest changing the name of the test or splitting it into two separate methods.